src/Repository/Criteria/FindLineItemCriteria.php

```diff
@@ -22,15 +22,17 @@
 
 namespace OAT\SimpleRoster\Repository\Criteria;
 
+use Symfony\Component\Uid\UuidV6;
+
 class FindLineItemCriteria
 {
-    /** @var int[] */
+    /** @var UuidV6[] */
     private $lineItemIds = [];
 
     /** @var string[] */
     private $lineItemSlugs = [];
 
-    public function addLineItemIds(int ...$lineItemIds): self
+    public function addLineItemIds(UuidV6 ...$lineItemIds): self
```

Comment:
Just a stupid question:
why the method is `add...` instead of `set...`, since we are not adding inside, but setting?